controllers/istio_controller_test.go

```diff
@@ -295,3 +298,189 @@ func newCondition(conditionType v1.IstioConditionType, status bool, reason v1.Is
 		Reason: reason,
 	}
 }
+
+func TestApplyProfile(t *testing.T) {
+	resourceDir := t.TempDir()
+	profilesDir := path.Join(resourceDir, "v3.0", "profiles")
```

Comment:
This "v3.0" instance is not a problem, since it's only used within the TestApplyProfile function. It could be any string, as long as the same string is used in the testcases within TestApplyProfile.